### PR TITLE
feature: new settings page

### PR DESCRIPTION
**Pull request by bzp2010**
replace the original login page with the settings page.

### CODE REVIEW

**Comment by bzp2010** [comment]
Progress Overview

- [x] settings page migration
- [x] logout button modify
- MORE

File: src/locales/en-US/menu.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default {
   'menu.exception.trigger': 'Trigger',
   'menu.account': 'Account',
   'menu.account.center': 'Account Center',
-  'menu.account.settings': 'Account Settings',
+  'menu.account.settings': 'Account Settings1',
```
Comment (juzhiyuan) [inline src/locales/en-US/menu.ts:44]: Settings1?

File: src/locales/en-US/settingDrawer.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export default {
   'app.setting.fixedsidebar.hint': 'Works on Side Menu Layout',
   'app.setting.hideheader': 'Hidden Header when scrolling',
   'app.setting.hideheader.hint': 'Works when Hidden Header is enabled',
-  'app.setting.othersettings': 'Other Settings',
+  'app.setting.othersettings': 'Other Settings1',
```
Comment (juzhiyuan) [inline src/locales/en-US/settingDrawer.ts:25]: ,

File: src/pages/settings/style.less [hunk 3]
```diff
@@ -1,4 +1,14 @@
-@import '~antd/es/style/themes/default.less';
+@import '../../../node_modules/antd/es/style/themes/default.less';
```
Comment (juzhiyuan) [inline src/pages/settings/style.less:1]: ,

**Pull request merged**